cudaaligner/src/myers_gpu.cu

```diff
@@ -8,16 +8,18 @@
 * license agreement from NVIDIA CORPORATION is strictly prohibited.
 */
 
-#include "myers_gpu.cuh"
 #include <cassert>
 #include <climits>
 #include <vector>
 #include <numeric>
-#include <utils/signed_integer_utils.hpp>
-#include <utils/mathutils.hpp>
+
+#include <claragenomics/utils/signed_integer_utils.hpp>
+#include <claragenomics/utils/mathutils.hpp>
+#include <claragenomics/utils/cudautils.hpp>
+
+#include "myers_gpu.cuh"
```

Comment:
The header file (.h) belonging to the source file (.cpp/.cu) should be the first include, such that the compile fails if the header file is missing an include to be self-contained (e.g. you use vector<T> in your interface, but you didn't include <vector> in your header.)